README.md

```diff
@@ -78,225 +72,140 @@ maintained by [Parity Technologies](https://www.parity.io/). Source code availab
-  Polkadot parachains.
-- [Bridges](https://github.com/paritytech/parity-bridges-common) - A collection of tools for
-  cross-chain communication.
-- [Frontier](https://github.com/paritytech/frontier) - End-to-end Ethereum emulation for Substrate
-  chains.
+- [IntegriTEE](https://book.integritee.network/) - Trusted off-chain execution framework that uses [Intel SGX](https://en.wikipedia.org/wiki/Software_Guard_Extensions) trusted execution environments.
 - [Polkadot-JS](https://polkadot.js.org/) - Rich JavaScript API framework for front-end development.
-- [IntegriTEE](https://book.integritee.network/) - Trusted off-chain execution framework that uses
-  [Intel SGX](https://en.wikipedia.org/wiki/Software_Guard_Extensions) trusted execution
-  environments.
 
 ## Client Libraries
 
-- [`subxt`](https://github.com/paritytech/substrate-subxt) - Official Rust client.
-- [Substrate API Client](https://github.com/scs/substrate-api-client) - Rust client maintained by
-  [Supercomputing Systems AG](https://www.scs.ch/).
-- [Polkadot-JS API](https://github.com/polkadot-js/api/) - Semi-official JavaScript library for
-  Substrate-based chains.
-- [Python Substrate Interface](https://github.com/polkascan/py-substrate-interface) - Maintained by
-  [Polkascan Foundation](https://polkascan.org/).
-- [Go Substrate RPC Client](https://github.com/centrifuge/go-substrate-rpc-client/) - Maintained by
-  [Centrifuge](https://centrifuge.io/). 
-- [Substrate C++ API](https://github.com/usetech-llc/polkadot_api_cpp) - Maintained by
-  [Usetech](https://usetech.com/blockchain/).
-- [Substrate .Net API](https://github.com/usetech-llc/polkadot_api_dotnet) - Maintained by
-  Usetech.
-- [Substrate .NET Standard API](https://github.com/dotmog/SubstrateNetApi) - Used in
-  [nuget](https://www.nuget.org/packages/SubstrateNetApi/), and
-  [Unity 3D integration example](https://github.com/darkfriend77/Unity3DExample) - Maintained by
-  [DOTMog](https://dotmog.com/).
-- [Kotlin Substrate Client](https://github.com/NodleCode/substrate-client-kotlin) - Maintained by 
-  [Nodle.io](https://github.com/NodleCode)
+- [.Net API](https://github.com/usetech-llc/polkadot_api_dotnet) - Maintained by [Usetech](https://usetech.com/blockchain/).
+- [.NET Standard API](https://github.com/dotmog/SubstrateNetApi) - Used in [nuget](https://www.nuget.org/packages/SubstrateNetApi/), and [Unity 3D integration example](https://github.com/darkfriend77/Unity3DExample); Maintained by [DOTMog](https://dotmog.com/).
+- [`go-substrate-gen`](https://github.com/Aphoh/go-substrate-gen) - Generate go (de)serialization/client code from substrate metadata.
 - [`sube`](https://github.com/virto-network/sube) - Lightweight Rust client library and CLI with support for type information.
+- [`subxt`](https://github.com/paritytech/substrate-subxt) - Official Rust client.
+- [C++ API](https://github.com/usetech-llc/polkadot_api_cpp) - Maintained by Usetech.
```

Comment:
```suggestion
- [C++ API](https://github.com/usetech-llc/polkadot_api_cpp) - Maintained by [Usetech](https://usetech.com/blockchain/).
```

Reply:
This fails linting. You cannot reuse URLs. They already are called out above